arch/src/aarch64/uefi.rs

```diff
@@ -1,4 +1,5 @@
 // Copyright 2020 Arm Limited (or its affiliates). All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
To stay consistent with the rest of our license headers, please leave a blank line before adding the `SPDX` header.

Reply:
FWIW, there are many places need similar changes that I don't point to. Please update accordingly. Thank you.

Reply:
done